backend/main.py

```
@@ -1005,6 +978,122 @@ async def get_user_profile(user_id: int, db: Session = Depends(get_db)):
     
     return user
 
+# --- USER SESSION MANAGEMENT ---
+@app.post("/users/activity")
+async def track_user_activity(
+    request: dict,
```

Ruff (0.14.1)

Unused function argument: `request`

(ARG001)